ipn/ipnlocal/serve.go

```diff
@@ -750,14 +750,22 @@ func (b *LocalBackend) serveWebHandler(w http.ResponseWriter, r *http.Request) {
 	http.Error(w, "empty handler", 500)
 }
 
+// stripMountPointFromError removes the root directory from the error message
+// to avoid exposing the local filesystem path.
+func stripMountPointFromError(err error, fileOrDir string) string {
```

Comment:
Given my message below, this function might not be needed. If we stick with it, it's probably better to have it take a string so that it doesn't panic if someone accidentally passed it a nil-error. Otherwise, it should check for nil for safety. 

```suggestion
func stripMountPointFromError(mountPoint, fileOrDir string) string {
```